pkg/users/index.html

```diff
@@ -128,6 +128,18 @@ <h1 translatable="yes">Account not available or cannot be edited.</h1>
                  data-target="#password-expiration"></a>
             </td>
           </tr>
+          <tr id="session_recoding_tr">
```

Comment:
To avoid flicker, please hide this by default, and show it once you determine that it is supported.